### ISSUE TITLE
Could not load file or assembly 'NuGet.Versioning

### ISSUE THREAD

**Issue by brwilkinson** [PowerShell/PSResourceGet#350]
#### Install-PSResource : Could not load file or assembly 'NuGet.Versioning, Version=5.7.0.7, Culture=neutral, PublicKeyToken=31bf3856ad364e35' or one of its dependencies. The system cannot find the file specified.

```powershell
gmo *get*

ModuleType Version Name          ExportedCommands
---------- ------- ----          ----------------
Script     3.0.0   PowerShellGet {Find-PSResource, Get-PSResource, Get-PSResourceRepository, Install-PSResource...}

$PSVersionTable

Name                           Value
----                           -----
PSVersion                      5.1.21318.1000


Install-PSResource -name posh-git
Install-PSResource : Could not load file or assembly 'NuGet.Versioning, Version=5.7.0.7, Culture=neutral, PublicKeyToken=31bf3856ad364e35' or one of its dependencies. The system cannot
find the file specified.
At line:1 char:1
+ Install-PSResource -name posh-git
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Install-PSResource], FileNotFoundException
    + FullyQualifiedErrorId : System.IO.FileNotFoundException,Microsoft.PowerShell.PowerShellGet.Cmdlets.InstallPSResource
```

I saw some other references to this e.g.: https://github.com/Azure/azure-functions-host/issues/6650

**Comment by SydneyhSmith** [comment]
Thanks @brwilkinson this is an interesting issue...are you using Azure Functions? We packaged the nuget assemblies in the latest version of PowerShellGet, so re-installing the module may help...

**Comment by brwilkinson** [comment]
Closing this issue, since reinstalling PowerShellGet 3.0.0-beta10 has resolved this.

Not sure why all files weren't originally downloaded.

For some reason the Modules\PowerShellGet\3.0.0\net472 directory wasn't downloaded.

![image](https://user-images.githubusercontent.com/3605266/111826930-814a2200-88a6-11eb-93af-795c45d62ad4.png)

I actually tested it again and found it didn't get downloaded 1 of the 3 times that i tested. 

Whatever that was is unrelated to this issue.
